gatsby-config.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
       options: {
         extensions: ['.mdx', '.md', '.markdown'],
         defaultLayouts: {
-          default: require.resolve('./src/components/layout.js'),
+          default: require.resolve('./src/components/Layout.js'),
```

Comment:
Actually, I think we should keep lower case for this `layout` component (as default Gatsby does)
We renamed it when started to use Gatsby